src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -798,6 +798,7 @@ void AddLauncherIntentElements (XElement activity)
 				return;
 
 			var filter = new XElement ("intent-filter");
+			activity.Add (new XAttribute (androidNs + "exported", "true"));
```

Comment:
Do we want to add this for all Android versions or should we only be doing it for Android 12 + ?

Reply:
It does not appear that [`//activity/@android:exported` attribute](https://developer.android.com/guide/topics/manifest/activity-element#exported) has a minimum API level. Since this is a security feature, we're probably good to set this across the board?

Usually the way new attributes work in `AndroidManifest.xml`, is that older Android versions just ignore the values they don't know about.

Reply:
oh okay awesome then ! thank you